glossary.tex

```diff
@@ -1,31 +1,31 @@
-\newglossaryentry{Cluster}
+\newglossaryentry{cluster}
 {
-  name={Cluster},
+  name={cluster},
   description={A group of compute nodes},
 }
-\newglossaryentry{Compute-Node}
+\newglossaryentry{compute-node}
 {
-  name={Compute Node},
+  name={compute node},
   description={The computational units on which batch or interactive jobs are processed. A compute node is pretty much comparable to a single personal computer. It contains one or more sockets, each holding a single CPU. Some nodes also contain one or more GPGPUs. The compute node is equipped with memory (RAM) that is accessible by all its CPUs},
 }
-\newglossaryentry{Core}
+\newglossaryentry{core}
 {
-  name={Core},
+  name={core},
   description={An individual compute unit inside a CPU. A CPU typically contains one or more cores},
 }
 \newglossaryentry{CPU}
 {
   name={CPU},
   description={A central processing unit. A CPU is a consumable resource. A compute node typically contains one or more CPUs},
 }
-\newglossaryentry{Distributed-memory-system}
+\newglossaryentry{distributed-memory-system}
 {
-  name={Distributed memory system},
+  name={distributed memory system},
   description={Computing system consisting of many compute nodes connected by a network, each with their own memory. Accessing memory on a neighbouring node is possible but requires explicit communication},
 }
-\newglossaryentry{Flops}
+\newglossaryentry{FLOPS}
 {
-  name={Flops},
+  name={FLOPS},
   description={Floating-point Operations Per second},
```

Comment:
FLOPS is short for "Floating-point Operations Per second", i.e. the number of (floating-point) computations that a processor can perform per second.